### PR TITLE
Making excluded tests from` test/mri/ruby/test_env.rb` pass

### PR DESCRIPTION
**Pull request by edipofederle**
Making excluded tests from` test/mri/ruby/test_env.rb` pass.

` ./bin/jruby test/mri/runner.rb  test/mri/ruby/test_env.rb `

### CODE REVIEW

**Review by enebo — CHANGES_REQUESTED** [review]
@edipofederle It is great to see you tackling excluded tests in MRI.  I think we have lots of broken corner cases like this which need fixing.

File: core/src/main/java/org/jruby/RubyGlobal.java [hunk 1]
```diff
@@ -730,6 +733,16 @@ private IRubyObject case_aware_op_aset(ThreadContext context, IRubyObject key, f
             return value;
         }
 
+        private static void checkEnvValibility(Ruby runtime, IRubyObject str, String type) {
+            if(!str.convertToString().getByteList().getEncoding().isAsciiCompatible()){
```
[the remark this answers is not in the document]
Comment (edipofederle) [reply core/src/main/java/org/jruby/RubyGlobal.java:737]: @enebo yet, I will do it, and also fix the rest of excluded tests for ENV, since is related with it.

**Comment by enebo** [comment]
Thanks @edipofederle!
